sale_order_invoice_amount/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+The purpose of this module is to add two fields in the sale order model:
+* invoiced_amount: total invoiced amount.
```

Comment:
Before starting bullet lists you need to add new blank line, otherwise RST will not be displayed correctly.

Reply:
this is how it looks now: 
![image](https://user-images.githubusercontent.com/23449160/114864717-9ce71080-9df1-11eb-9cea-59a6051b76ce.png)